src/admin/dumb_components/custom_email_form.js

```diff
@@ -1,18 +1,19 @@
 const React = require('react')
+import preview_custom_email from '../dumb_components/preview_custom_email.js'
 
-export default ({custom_emails: { members }, submit_custom_email }) => {
+export default ({ custom_emails: { members, preview, mode }, preview_custom, ...props}) => {
   const onSubmit = e => {
     e.preventDefault()
-    submit_custom_email(members, e.target)
+    preview_custom([ e.target[0].value, e.target[1].value ])
   }
-  return (
-    <div className='custom-email-container'>
-      <p><i>Write out your custom email.</i></p>
-      <form onSubmit={onSubmit}>
-        <input type='text' className='custom-email-subject' placeholder='Subject'></input>
-        <textarea className='custom-email' placeholder='Email body' />
-        <button type='submit'>Submit</button>
-      </form>
-    </div>
-  )
+  return mode === 'PREVIEW_CUSTOM'
+    ? preview_custom_email({ members, preview, props })
+    : <div className='custom-email-container'>
```

Comment:
I think this is too large a component to be inside a ternary. Maybe extract it out?